src/frontend/package.json

```
@@ -19,6 +19,10 @@
     "@opentelemetry/exporter-trace-otlp-grpc": "0.33.0",
     "@opentelemetry/exporter-trace-otlp-http": "0.33.0",
     "@opentelemetry/instrumentation": "0.33.0",
+    "@opentelemetry/resource-detector-alibaba-cloud": "^0.27.3",
+    "@opentelemetry/resource-detector-aws": "^1.2.1",
+    "@opentelemetry/resource-detector-container": "^0.2.1",
+    "@opentelemetry/resource-detector-gcp": "^0.27.4",
```

Consistent version pinning
```suggestion
    "@opentelemetry/resource-detector-alibaba-cloud": "0.27.3",
    "@opentelemetry/resource-detector-aws": "1.2.1",
    "@opentelemetry/resource-detector-container": "0.2.1",
    "@opentelemetry/resource-detector-gcp": "0.27.4",
```